src/main/java/net/minecraftforge/client/event/RegisterParticleProvidersEvent.java

```diff
@@ -39,38 +40,109 @@ public RegisterParticleProvidersEvent(ParticleEngine particleEngine)
     }
 
     /**
-     * Registers a ParticleProvider for a non-json-based ParticleType.
-     * These particles do not receive a list of texture sprites to use for rendering themselves.
+     * <p>Registers a ParticleProvider for a non-json-based ParticleType.
+     * These particles do not receive a list of texture sprites to use for rendering themselves.</p>
+     * 
```

Comment:
I believe there shouldn't be whitespace after the asterisk on blank javadoc comment lines...?